config/routes.rb

```diff
@@ -60,7 +61,8 @@
     resource :privacy_policy, controller: 'about/privacy_policy', path: 'privacy-policy', only: [:show]
     resource :travel_calculator, controller: 'travel_calculator', path: 'travel-calculator', only: [:show]
 
-    resources :lifestyle_footprints, path: 'calculator', only: [:new, :create], path_names: { new: '' }
+    resources :lifestyle_footprints, path: 'calculator', only: [:new, :create, :destroy], path_names: { new: '' }
+    get 'calculator/result/:id', to: 'lifestyle_footprints#show', as: 'lifestyle_footprints_result'
```

Comment:
Maybe this is me being deep in la la Rails land, but I would prefer this to clarify that this is really the same resource, but with different paths:

```suggestion
    resources :lifestyle_footprints, path: 'calculator', only: [:new, :create], path_names: { new: '' } do
      collection do
        resources :lifestyle_footprints, path: 'result', only: [:show, :destroy]
      end
    end
```

(This also changes the path helper from `lifestyle_footprint_result` to `lifestyle_footprint` the same way that standard `show` actions get the resource in singular as their names.)

Reply:
If you agree to move the index action here as well, this is what I'd suggest:

```suggestion
    resources :lifestyle_footprints, path: 'calculator', only: [:new, :create], path_names: { new: '' } do
      collection do
        resources :lifestyle_footprints, path: 'results', only: [:index, :show, :destroy]
      end
    end
```

(Add the plural `s` to `results` and the action itself.)